app/resources/views.py

```diff
@@ -45,18 +46,76 @@ def add():
                             user_id=int(current_user.get_id()))
         db.session.add(resource)
         db.session.commit()
-        return redirect(url_for('resources.index'))
+        return redirect(url_for('resources.show', resource_id=resource.id))
     return render_template('resources/add_resource.html', form=form)
 
 
 @resources.route('/resource/<int:resource_id>')
 @login_required
-def show_resource(resource_id):
-    """
-    Show the resource with the given id, the id is an integer.
-    """
+def show(resource_id):
```

Comment:
Is there anything wrong with naming this function `view_resource` (the same as the html file)